### PR TITLE
version 0.6.0

### PR DESCRIPTION
**Pull request by gcanti**
# 0.6.0

- **Breaking Change**
  - upgrade to `fluture@12.0.1`

### CODE REVIEW

File: src/Future.ts [hunk 1]
```diff
@@ -25,24 +22,13 @@ export type URI = typeof URI
...
-    Future.chainRec<E, A, B>(
-      (next, done, value) =>
-        f(value).map(e =>
-          pipe(
-            e,
-            fold<A, B, Next<A> | Done<B>>(next, done)
-          )
-        ),
-      a
-    ),
```
Comment (Avaq) [inline src/Future.ts:46]: Why was `chainRec` removed?

Comment (gcanti) [reply src/Future.ts:46]: Oh I'm just not sure how it can be implemented now, I need some time to figure it out

Comment (Avaq) [reply src/Future.ts:46]: I'll give it a try.

Comment (Avaq) [reply src/Future.ts:46]: #14

Comment (Avaq) [reply src/Future.ts:46]: I threw it together to give you the general idea. I probably missed some stuff. If you want I can have a better look later when I have more time.

File: test/index.ts [hunk 2]
```diff
@@ -20,39 +17,36 @@ describe('Fluture bindings', () => {
...
+    assert.deepEqual(xs, [1, 2])
   })
 
-  it('should export a ChainRec instance', () => {
-    return future
-      .chainRec(1, a => of(right(a)))
-      .promise()
-      .then(n => {
-        assert.strictEqual(n, 1)
-      })
```
Comment (Avaq) [inline test/index.ts:38]: Should this removal be reverted?

**Comment by Avaq** [comment]
I see you've amended my commit with the necessary fixes. You probably also want to reword it, as the chainRec implementation is no longer a "concept", as the commit message suggests.

**Review by Avaq — APPROVED** [review]
LGTM

**Comment by gcanti** [comment]
@Avaq thank you for reviewing!

**Pull request merged**